### DISCUSSION TITLE
How do I embed a revealjs presentation as a Github submodule in a Quarto list?

### DISCUSSION THREAD

**Discussion by chrisvoncsefalvay** [quarto-dev/quarto-cli#7583] [Q&A] [answered]
### Description

I have a Quarto blog that lists, among others, various publications of mine. I'd like to embed a revealjs presentation, made with Quarto, but the presentation is in a separate Github repo. I want it to be embedded as a Github submodule so I can propagate changes down. Is there a good, canonical way to do this?

**Comment by mcanouil** [comment, marked as answer]
In general, git submodules are discouraged for nearly everything ...

This being said, depending on what "embed" means, you could simply in a listing (is that what yoj mean by "Quarto list?) set the href to the external repository or use iframe.

This is entirely up to you and I don't think Quarto has or should have any guidelines here.

Edit: I invite you to search though the discussions (you can use the labels and/or keywords) as embedded presentation in website/blog has been asked several times already and featured several answers.

**Reply by chrisvoncsefalvay** [reply]
Thanks, so a listing can draw on files outside the current Quarto blog/website?

**Reply by mcanouil** [reply]
You can target external HTML files.
There are plenty of meaning behind "embed".
Since I still don't know what you mean by that and/or what you want to achieve exactly, I can't help more than highlighting the documentation which shows how to use listings.
